apps/website/tsconfig.json

```diff
@@ -26,9 +27,10 @@
     "target": "ES2019",
     "declarationMap": true,
     "baseUrl": ".",
-    "types": ["mdx"],
-    "allowJs": true
+    "types": ["mdx", "@docusaurus/theme-classic"],
```

Comment:
Is this scalable? Do we always have to put everything in here?

Reply:
just for '@docusaurus' imports, they're provided virtually by docusaurus' webpack config

Reply:
I don't think so, but docusaurus has special magic types, maybe if we stop hoisting this type it will work?

Reply:
Tried to remove, tsc was not happy. Other types work fine.